vendor-overwrites/codemirror/addon/lint/css-lint.js

```diff
@@ -15,47 +15,85 @@
+    /* STYLISH: hack end */
 
-  var results = CSSLint.verify(text), messages = results.messages, message = null;
-  for ( var i = 0; i < messages.length; i++) {
-    message = messages[i];
+    var results = CSSLint.verify(text), messages = results.messages, message = null;
+    for ( var i = 0; i < messages.length; i++) {
+      message = messages[i];
 
-    /* STYLISH: hack start (part 2) */
-    if (message.type === 'warning') {
-       // @font-face {font-family: 'Ampersand'; unicode-range: U+26;}
-      if (message.message.indexOf('unicode-range') !== -1) {
-        continue;
-      }
-      else if ( // color: hsl(210, 100%, 2.2%); or color: hsla(210, 100%, 2.2%, 0.3);
-        message.message.startsWith('Expected (<color>) but found \'hsl') &&
-        /hsla?\(\s*(-?\d+)%?\s*,\s*(-?\d+)%\s*,\s*(-?\d+|-?\d*.\d+)%(\s*,\s*(-?\d+|-?\d*.\d+))?\s*\)/.test(message.message)
-      ) {
-        continue;
+      /* STYLISH: hack start (part 2) */
+      if (message.type === 'warning') {
+         // @font-face {font-family: 'Ampersand'; unicode-range: U+26;}
+        if (message.message.indexOf('unicode-range') !== -1) {
+          continue;
+        }
+        else if ( // color: hsl(210, 100%, 2.2%); or color: hsla(210, 100%, 2.2%, 0.3);
+          message.message.startsWith('Expected (<color>) but found \'hsl') &&
+          /hsla?\(\s*(-?\d+)%?\s*,\s*(-?\d+)%\s*,\s*(-?\d+|-?\d*.\d+)%(\s*,\s*(-?\d+|-?\d*.\d+))?\s*\)/.test(message.message)
+        ) {
+          continue;
+        }
+        //
       }
-      //
+      /* STYLISH: hack end */
+
+      var startLine = message.line -1, endLine = message.line -1, startCol = message.col -1, endCol = message.col;
+      found.push({
+        from: CodeMirror.Pos(startLine, startCol),
+        to: CodeMirror.Pos(endLine, endCol),
+        message: message.message,
+        severity : message.type
+      });
     }
-    /* STYLISH: hack end */
+  }
+  return found;
+});
 
-    var startLine = message.line -1, endLine = message.line -1, startCol = message.col -1, endCol = message.col;
-    found.push({
-      from: CodeMirror.Pos(startLine, startCol),
-      to: CodeMirror.Pos(endLine, endCol),
-      message: message.message,
-      severity : message.type
+CodeMirror.registerHelper("lint", "stylelint", function(text) {
+  let found = [];
+  window.stylelint = require('stylelint').lint;
+  if (window.stylelint) {
+    return BG.chromeLocal.getValue('editorStylelintRules').then((rules = stylelintDefaultConfig.rules) => {
+      // stylelintDefaultConfig stored in stylelint-config.js & loaded by edit.html
+      if (Object.keys(rules).length === 0) {
+        rules = stylelintDefaultConfig.rules;
+      }
+      return stylelint({
+        code: text,
+        config: {
+          syntax: stylelintDefaultConfig.syntax,
+          rules: rules
+        }
+      }).then(output => {
+        const warnings = output.results.length ? output.results[0].warnings : [],
+          len = warnings.length;
```

Comment:
a separate `const` for a variable with initializer should be used as per our eslint config, but see my comment above